docs/docsite/rst/playbook_guide/playbooks_module_defaults.rst

```diff
@@ -175,4 +128,52 @@ This group can now be used in a playbook like this:
+      - another.collection.another_module:
+
+For historical reasons and backwards compatibility, there are some special groups:
+
++---------+--------------------------------------------------------------------------------------------------------------------+
+| Group   | Extended module group                                                                                              |
++=========+====================================================================================================================+
+| aws     | amazon.aws.aws and community.aws.aws                                                                               |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| azure   | azure.azcollection.azure                                                                                           |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| gcp     | google.cloud.gcp                                                                                                   |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| k8s     | community.kubernetes.k8s, community.general.k8s, community.kubevirt.k8s, community.okd.k8s, and kubernetes.core.k8s|
++---------+---------------------------+----------------------------------------------------------------------------------------+
+| os      | openstack.cloud.os                                                                                                 |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| acme    | community.crypto.acme                                                                                              |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| docker* | community.general.docker and community.docker.docker                                                               |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| ovirt   | ovirt.ovirt.ovirt and community.general.ovirt                                                                      |
++---------+--------------------------------------------------------------------------------------------------------------------+
+| vmware  | community.vmware.vmware                                                                                            |
++---------+--------------------------------------------------------------------------------------------------------------------+
+
+* Check out the community.docker documentation which Docker modules are included in the ``docker`` defaults group.
```

Comment:
Looks good to me.  My only suggestion is this should be a more generic statement since all collections are responsible for the list of modules/action plugins in the extended groups. The hardcoded list in ansible-core was removed for 2.10 (in https://github.com/ansible/ansible/pull/69919).

```suggestion
* Check out the documentation for the collection or its meta/runtime.yml to see which action plugins and modules are included in the group.
```

Reply:
In that case, the `*` next to `docker` should be removed (and probably also the one in front of this sentence).